yosh/shell.py

```diff
@@ -16,8 +16,55 @@
 def tokenize(string):
```

Comment:
If `el` (or `token[i]`) is `"a="`, 

then, `el.find('=')` will return `1` and `len(token[i])` will return `2`, isn't it? 

In other words, if `el` and `token[i]` are the same thing, `.find()` will never be equal to `len()`. (Only case that will be equal is that it's an empty string and .find(''). However, an empty string will be not included in our tokens)